### PR TITLE
Fix spelling errors in binaries.

### PR DESCRIPTION
**Pull request by mzf-guest**
Hello,

here the patch from the Debian package that removes common spelling errors in the generated binaries.

Best,

François

### CODE REVIEW

File: src/bamglib/Mesh2.cpp [hunk 1]
```diff
@@ -4547,7 +4547,7 @@ namespace bamg {
     area *= 3;
     gammamn = sqrt(gammamn);
     gammamx = sqrt(gammamx);
-    cout << "  -- adaptmesh Regulary:  Nb triangles " << nt << " , h  min " << hmin << " , h max "
+    cout << "  -- adaptmesh Regularly:  Nb triangles " << nt << " , h  min " << hmin << " , h max "
```
Comment (prj-) [inline src/bamglib/Mesh2.cpp:4550]:
```suggestion
    cout << "  -- adaptmesh statistics:  Nb triangles " << nt << " , h  min " << hmin << " , h max "
```

File: src/fflib/lgfem.cpp [hunk 2]
```diff
@@ -1157,7 +1157,7 @@ bool BuildPeriodic2(int nbcperiodic, Expression *periodic, const Mesh &Th, Stack
             else {
               cout << ie2 << " ~ " << im0->second[0] << " " << im0->second[1] << ", "
                    << im1->second[0] << " " << im1->second[1] << endl;
-              ExecError("periodic: Sorry one egde is losted ");
+              ExecError("periodic: Sorry one edge is losted ");
```
Comment (prj-) [inline src/fflib/lgfem.cpp:1160]:
```suggestion
              ExecError("periodic: Sorry one edge is lost");
```

**Review by prj- — CHANGES_REQUESTED** [review]
Please revert the changes in `readme/HISTORY` and `readme/HISTORY_BEFORE_2005`. These are legacy files that should not be updated anymore. Thank you for your contribution.

File: src/fflib/lgfem.cpp (hunk 2 above)
Comment (mzf-guest) [reply src/fflib/lgfem.cpp]: Done!

File: src/bamglib/Mesh2.cpp (hunk 1 above)
Comment (mzf-guest) [reply src/bamglib/Mesh2.cpp]: Done!

**Comment by mzf-guest** [comment]
> Please revert the changes in `readme/HISTORY` and `readme/HISTORY_BEFORE_2005`. These are legacy files that should not be updated anymore. Thank you for your contribution.

Done!

**Comment by prj-** [comment]
Thank you for your contribution.

**Pull request merged**